draft-mohan-asdf-sdf-protocol-mapping.md

```diff
@@ -420,6 +543,8 @@ Following protocol mappings are described in this document:
 |--------------|-----------------------------|---------------------------------------------|-----------------|
 | ble          | Bluetooth Low Energy (BLE)  | Protocol mapping for BLE devices            | This document   |
 | zigbee       | Zigbee                      | Protocol mapping for Zigbee devices         | This document   |
+| http         | HTTP                        | Protocol mapping for HTTP IP-based devices  | This document   |
+| coap         | CoAP                        | Protocol mapping for CoAP IP-based devices  | This document   |
```

Comment:
Since (at least theoretically) you could also use CoAP over something like [Bluetooth GATT](https://datatracker.ietf.org/doc/html/draft-amsuess-core-coap-over-gatt-07) or even [SMS](https://datatracker.ietf.org/doc/html/draft-becker-core-coap-sms-gprs-06), maybe we could also omit "IP-based" here.

```suggestion
| coap         | CoAP                        | Protocol mapping for CoAP-based devices  | This document   |
```